package.json

```diff
@@ -35,6 +36,7 @@
     "eslint-plugin-import": "2.23.4",
     "get-image-colors": "4.0.0",
     "jest": "27.0.6",
+    "jimp": "0.16.1",
```

Comment:
This dependency is used only in tests (`get-image-colors` too), why they are not inside devDependenices?

Reply:
it is in `devDependencies`. I guess that GitHub collapsed the difference that it looks like it's in `dependencies`